rules-reviewed/eap7/eap71/hibernate51-53.windup.xml

```diff
@@ -0,0 +1,155 @@
+        <rule id="hibernate51-53-00300">
+            <when>
+            </when>
+            <perform>
+            </perform>
+            <where param="">
+                <matches pattern="" />
+            </where>
+        </rule>
+        -->
+        <!--https://issues.jboss.org/browse/WINDUPRULE-380
+        <rule id="hibernate51-53-00400">
+            <when>
+            </when>
+            <perform>
+            </perform>
+        </rule>
+        -->
+        <!--https://issues.jboss.org/browse/WINDUPRULE-381
+        <rule id="hibernate51-53-00500">
+            <when>
+            </when>
+            <perform>
+            </perform>
+            <where param="">
+                <matches pattern="" />
+            </where>
+        </rule>
+        -->
+        <!--https://issues.jboss.org/browse/WINDUPRULE-382
+        <rule id="hibernate51-53-00600">
+            <when>
+            </when>
+            <perform>
+            </perform>
+        </rule>
+        -->
+        <!--https://issues.jboss.org/browse/WINDUPRULE-383
+        <rule id="hibernate51-53-00700">
+            <when>
+            </when>
+            <perform>
+            </perform>
+            <where param="">
+                <matches pattern="" />
+            </where>
+        </rule>
+        -->
+        <!--https://issues.jboss.org/browse/WINDUPRULE-384
+        <rule id="hibernate51-53-00800">
+            <when>
+            </when>
+            <perform>
+            </perform>
+        </rule>
+        -->
+        <!--https://issues.jboss.org/browse/WINDUPRULE-385
+        <rule id="hibernate51-53-00900">
+            <when>
+            </when>
+            <perform>
+            </perform>
+            <where param="">
+                <matches pattern="" />
+            </where>
+        </rule>
+        -->
+        <!--https://issues.jboss.org/browse/WINDUPRULE-386
+        <rule id="hibernate51-53-01000">
+            <when>
+            </when>
+            <perform>
+            </perform>
+        </rule>
+        -->
+        <!--https://issues.jboss.org/browse/WINDUPRULE-387
+        <rule id="hibernate51-53-01100">
+            <when>
+            </when>
+            <perform>
+            </perform>
+            <where param="">
+                <matches pattern="" />
+            </where>
+        </rule>
+        -->
+        <!--https://issues.jboss.org/browse/WINDUPRULE-388-->
+        <rule id="hibernate51-53-01200">
+            <when>
+                <javaclass references="org.hibernate.{className}.getType({*})">
+                </javaclass>
+            </when>
+            <perform>
+                <hint title="Hibernate 5.3 Compatibility Issues - org.hibernate.query.{className}.getType() renamed" effort="1" category-id="mandatory">
+                    <message>Any references to org.hibernate.{className}.getType() must be replaced with org.hibernate.{className}.getHibernateType().</message>
```

Comment:
Added the *`* character to mark the pieces of code about the fully qualified class names
```suggestion
                    <message>Any references to `org.hibernate.{className}.getType()` must be replaced with `org.hibernate.{className}.getHibernateType()`.</message>
```